### PR TITLE
baseball bat with a Toolbox

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request

image

adds a new weapons

## Why It's Good For The Game

this is a alternative to the spear,  it have more damage than a normal bat but less damage the spear, still have the knock 

## Changelog
:cl:
add: Added a toolbox to a baseball bat

/:cl:

### CODE REVIEW

File: ModularBungalow/meele_weapons.dm [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	name = "Baseball bat with a toolbox"
+	desc = "There ain't a skull in the league that can withstand a toolbox."
+	icon = 'ModularBungalow/zbungalowicons/weapons/melee.dmi'
+	icon_state = "baseball_toolbox"
+	inhand_icon_state = "baseball_toolbox"
+	lefthand_file = 'ModularBungalow/zbungalowicons/weapons/melee_lefthand.dmi'
+	righthand_file = 'ModularBungalow/zbungalowicons/weapons/melee_righthand.dmi'
+	force = 11
+	wound_bonus = -5
+	throwforce = 12
```
Comment (SgtHunk) [inline ModularBungalow/meele_weapons.dm:11]: you don't need this, parent has it

File: ModularBungalow/meele_weapons.dm [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	hitsound = 'sound/weapons/smash.ogg'
+	attack_verb_continuous = list("beats", "smacks", "robusts")
+	attack_verb_simple = list("beat", "smack", "robust")
+	custom_materials = list(/datum/material/wood = MINERAL_MATERIAL_AMOUNT * 2.5)
+	w_class = WEIGHT_CLASS_HUGE
```
Comment (SgtHunk) [inline ModularBungalow/meele_weapons.dm:16]: you don't need this, parent has it

**Review by SgtHunk — CHANGES_REQUESTED** [review]
Should be good after addressing requests, also the .dm file name says meele instead of melee, you should fix it for spelling's sake.

File: ModularBungalow/meele_weapons.dm (hunk 1 above)
Comment (DaddyIssues98) [reply ModularBungalow/meele_weapons.dm:11]: different stats, except the throw force, that i will delete. thanks anyway for the other stuff. luv u

**Pull request merged**